source/documentation/get-report-reconciliation-results.html.md.erb

```diff
@@ -0,0 +1,28 @@
+---
+title: Get report reconciliation results | ISA Returns API service guide
+weight: 7
+---
+
+# Get report reconciliation results
+
+After the results summary is available, you can retrieve the reconciliation results for the monthly report. These show which records were accepted and which were rejected, including error messages where applicable.
```

Comment:
I may be incorrect here, or this may still be up in the air - is it the case that a report is available if there were no errors with the submission? I thought there was only a report in case of errors/issues.

Reply:
Good question. I suppose that if there were no reconciliation/validation results for a monthly report, a 404 would be returned if you try to retrieve the reconciliation results. Is a question for @RobertLaverick?